src/context/json.ts

```diff
@@ -9,10 +17,23 @@ import { ResponseTransformer } from '../response'
  */
 export const json = <BodyType>(
   body: BodyType,
+  {
+    merge = defaultJSONContextOptions.merge,
```

Comment:
So that if we allow more options in the future and one is `false` we don't overwrite it with others.